dist/common/scripts/scylla_tune_sched

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+#
+# Copyright 2023-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import os
+import sys
+import yaml
+
+PROCFS = '/proc/sys/kernel'
+DEBUGFS = '/sys/kernel/debug'
+
+TUNE_PARAMS = {
+    # Prevent auto-scaling from doing anything to our tunables
+    'sched.tunable_scaling': 0,
+
+    # Preempt sooner (For CFS, only available on <linux-6.6)
+    'sched.min_granularity_ns': 500000,
+
+    # Preempt sooner  (For EEVDF, only available on >=linux-6.6)
+    'sched.base_slice_ns': 500000,
```

Comment:
@xemul we should examine EEVDF and see if we can apply any lessons to the Seastar scheduler